tests/qml/tst_GleanEvents.qml

```diff
@@ -0,0 +1,144 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's just remove these tests for now and work on proper tests once we have looked into how to do that :)

Reply:
Removed! I believe that's the last outstanding conversation - let me know if you want any other tweaks.